scripts/azure-pipelines.yml

```diff
@@ -33,7 +33,7 @@ variables:
   THROW_ON_TEST_FAILURE: true
   NUGET_DIFF_PRERELEASE: false
   ENABLE_CODE_COVERAGE: true
-  EMSCRIPTEN_VERSION: 1.39.18
+  EMSCRIPTEN_VERSION: 1.40.0
```

Comment:
What version of emscripten should and can be used? I know there is no guarantee of compatibility between versions... Does this change with 1.40.x?

Reply:
At this point, it has to be synchronized with the emscripten version specified here: https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/3154b8ec7396c136c1b919922678c01b1e53cf92/src/Uno.Wasm.Bootstrap/Constants.cs#L20 That being said, the bootstrapper's latest build automatically uses its own copy of an embedded emscripten, so steps for installing emscripten should now be safely removed.

It seems that mono version used by 1.4-dev is stable enough, so we'll probably make a stable out of it. Next bits of mono will use emscripten 2.0.x.

Could be that at some point we'll need to make multiple versions of the binaries with different versions of emscripten...